src/main/resources/templates/notifications/upload-notification-attachment.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en" th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'actions'},breadcrumbs=~{::#breadcrumbs})}" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>Provide Documents or Evidence</title>
+</head>
+<body>
+
+<div class="govuk-width-container">
+  <div id="breadcrumbs">
+    <a
+        th:href="@{/notifications/{notification_id}/provide-documents-or-evidence(notification_id=${notificationId})}"
+       class="govuk-back-link">Return to Provide Documents or Evidence</a>
+  </div>
+  <div id="main-content">
+    <h1 class="govuk-heading-l">Upload Postal Documents or Evidence</h1>
+    <form
+        action="#" method="post"
+        th:action="@{/notifications/{notification_id}/attachments/upload(notification_id=${notificationId})}" th:object="${attachmentUploadFormData}" enctype="multipart/form-data">
+      <div th:replace="~{partials/error-summary :: error-summary}"></div>
+      <div th:if="${attachmentUploadFormData.sendBy.code} == 'E'" class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-visually-hidden">Warning</span>
+          We can only accept <span th:text="${validExtensions}"/> files that are less than <span th:text="${maxFileSize}"/>.
+        </strong>
+      </div>
+      <p class="govuk-body-l">
+        Please provide details of the document you will be <span
+          th:text="${attachmentUploadFormData.sendBy.code} == 'P' ? 'posting' : 'uploading'"></span>.
```

Comment:
When accessing the form data you can accesss the data like this:

th:text="*{sendBy.code} == 'P' ? 'posting' : 'uploading'"></span>.

Just shortens the template. this works by using the form - th:object="${attachmentUploadFormData}"

Reply:
Thanks, updated.